lib/kernel.ts

```diff
@@ -314,6 +330,8 @@ export const unsafeUpsertContract = async (
 export class Kernel {
 	private backend: DatabaseBackend;
 	private sessions?: { admin: string };
+	relationshipsStream: any;
```

Comment:
This can be properly typed as 
```suggestion
	relationshipsStream: Stream;
```

Reply:
👍